mistletoe/html_renderer.py

```diff
@@ -40,38 +42,38 @@ def __exit__(self, *args):
         super().__exit__(*args)
         html._charref = self._stdlib_charref
 
-    def render_to_plain(self, token):
+    def render_to_plain(self, token) -> str:
         if hasattr(token, 'children'):
             inner = [self.render_to_plain(child) for child in token.children]
             return ''.join(inner)
         return self.escape_html(token.content)
 
-    def render_strong(self, token):
+    def render_strong(self, token: span_token.Strong):
```

Comment:
Good point. I added `-> str` hints to the `render_*()` methods.

I am a bit confused by that TypeError you mentioned. Python by default does not check type hints. You need a static type checker like mypy or pyright for that.

Reply:
> Good point. I added `-> str` hints to the `render_*()` methods.

Thanks.

> I am a bit confused by that TypeError you mentioned. Python by default does not check type hints. You need a static type checker like mypy or pyright for that.

I see. The `TypeError` is a _runtime_ error which you get when you try to return a number instead of a string from some render method - which I temporarily did for testing this behavior. I hope it's clear now. :)